deepchecks/nlp/input_validations.py

```diff
@@ -231,3 +235,197 @@ def compare_dataframes(
         difference = None
 
     return DataframesComparison(common, difference)
+
+
+def _validate_text_classification(
+    *,
```

Comment:
again why we need arbitrary arg?

Reply:
> sorry, not sure whether I get your question correctly,
> `*` is used to define keyword-only parameters ([docs](https://peps.python.org/pep-0570/#keyword-only-arguments)),
> parameters that can be passed only as keywords
> 
> IMO it will increase readibility